src/appleseed/foundation/meta/tests/test_zip.cpp

```diff
@@ -81,6 +81,40 @@ TEST_SUITE(Foundation_Utility_Zip)
         }
     }
 
+    TEST_CASE(ZipUnzip_UnzipsCorrectDirectory)
+    {
+        const string InitialDirectory = "unit tests/inputs/test_zip";
+        const string TargetZip = "unit tests/outputs/test_zip.zip";
+        const string TargetDirectory = "unit tests/outputs/test_zip";
+
+        try
+        {
+            ASSERT_TRUE(bf::exists(InitialDirectory));
+            ASSERT_FALSE(bf::exists(TargetZip));
+            ASSERT_FALSE(bf::exists(TargetDirectory));
+
+            zip(TargetZip, InitialDirectory);
+            unzip(TargetZip, TargetDirectory);
+
+            const set<string> ExpectedFiles = recursive_ls(InitialDirectory);
```

Comment:
Previously you said that constants should be LikeThat. Why it's not the case here?